app/views/pages/home/_footer_sub_hero_content.es.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2 class="heading5">
+  Vols contactar amb el Servei de Participació Ciutadana de L’Hospitalet de Llobregat? Pots esciure un correu electrònic a <a href="mailto:participacio.ciutadana@l-h.cat">participacio.ciutadana@l-h.cat</a> o be trucar al <strong>934026060</strong>.
```

Comment:
Text is in Catalan, but the file is for Spanish language

Reply:
oh